server/app.js

```diff
@@ -34,7 +34,9 @@ function createErrorHandler (response) {
 }
 if (process.env.FORCE_SSL && process.env.FORCE_SSL.toLowerCase() === 'true') {
   app.use((request, response, next) => {
-    if (request.secure || request.headers['x-forwarded-proto'] === 'https') {
+    if (request.secure || request.headers['x-forwarded-proto'] === 'https' || request.route === '/healthcheck') {
```

Comment:
Trying this new GitHub suggestion thing. Not sure how well it will work because multiple lines. Anyway! Did this work for you? It doesn’t work for me. `route` is a route object (not a URL path string) and represents the *currently matched* one, which I think doesn’t exist here since this is a universal middleware operating on all routes. I think you want `path`, but we should probably also handle the trailing slash:

```suggestion
  const healthcheckPath = /^\/healthcheck\/?$/;
  app.use((request, response, next) => {
    if (request.secure || request.headers['x-forwarded-proto'] === 'https' || healthcheckPath.test(request.path)) {
```

Reply:
Docs: http://expressjs.com/en/4x/api.html#req.route